packages/csvviewer-extension/src/index.ts

```diff
@@ -116,13 +116,14 @@ function activateCsv(
     );
 
     if (settingRegistry) {
-      toolbarFactory = createToolbarFactory(
+      toolbarFactory = createToolbarFactory({
         toolbarRegistry,
         settingRegistry,
-        FACTORY_CSV,
-        csv.id,
-        translator
-      );
+        factoryName: FACTORY_CSV,
+        pluginId: csv.id,
+        translator,
+        isPluginAvailable: (pluginId: string) => app.hasPlugin(pluginId)
```

Comment:
Maybe Lumino should also expose whether a plugin is *activated*? Not just whether it's in the `_pluginMap`.

In that case that function could be modified to also check for `app.isPluginActivated(pluginId)` (in case such function would exist in Lumino at some point).